src/TenorGuitar/Page.purs

```diff
@@ -0,0 +1,573 @@
+      -- audioCtx = unsafePartial (fromJust state.mAudioContext)
+      graphicsCtx <- H.liftEffect $ getContext2D canvas
+      -- _ <- H.liftEffect $ Drawing.render graphicsCtx chordDisplay
+      _ <- H.modify
+        ( \st -> st
+            { mGraphicsContext = Just graphicsCtx
+            , mCanvas = mCanvas
+            }
+        )
+      _ <- handleQuery (GetCanvasOffset unit)
+      _ <- handleQuery (DisplayFingering unit)
+      _ <- handleQuery (LoadInstruments unit)
+      pure unit
+    MouseDown cx cy -> do
+      state <- H.get
+      let
+        x = toNumber cx - state.canvasPosition.left
+        y = toNumber cy - state.canvasPosition.top
+      if (y > titleDepth) then do
+        let
+          fstring = fingeredString { x, y }
+        _ <- H.modify
+          ( \st -> st
+              { mouseDownFinger = Just fstring
+              , errorText = ""
+              , pitches = []
+              }
+          )
+        pure unit
+      else do
+        _ <- H.modify
+          ( \st -> st
+              { mouseDownFinger = Nothing
+              , pitches = []
+              }
+          )
+        pure unit
+    MouseUp cx cy -> do
+      state <- H.get
+      let
+        x = toNumber cx - state.canvasPosition.left
+        y = toNumber cy - state.canvasPosition.top
+      if (y > titleDepth) then do
+        let
+          fstring = fingeredString { x, y }
+        _ <- H.modify (\st -> st { mouseUpFinger = Just fstring, errorText = "" })
+        _ <- handleQuery (EditFingering unit)
+        pure unit
+      else do
+        _ <- H.modify (\st -> st { mouseUpFinger = Nothing })
+        pure unit
+    GetChordName name -> do
+      state <- H.get
+      let
+        newShape = state.chordShape { name = name }
+        newState = state
+          { chordShape = newShape
+          , errorText = ""
+          , pitches = []
+          }
+      _ <- H.put newState
+      _ <- handleQuery (DisplayFingering unit)
+      pure unit
+    GetFirstFretNumber numStr -> do
+      state <- H.get
+      let
+        fret = fromMaybe 0 $ fromString numStr
+        newShape = state.chordShape { firstFretOffset = fret }
+        newState = state { chordShape = newShape, errorText = "" }
+      _ <- H.put newState
+      _ <- handleQuery (DisplayFingering unit)
+      pure unit
+    ClearFingering -> do
+      _ <- H.modify
+        ( \st -> st
+            { chordShape = openStringChordShape
+            , errorText = ""
+            , pitches = []
+            }
+        )
+      _ <- handleQuery (DisplayFingering unit)
+      pure unit
+    GetImageScale scale -> do
+      _ <- H.modify (\st -> st { exportScale = scale })
+      pure unit
+    Export format -> do
+      state <- H.get
+      let
+        originalCanvas = unsafePartial (fromJust state.mCanvas)
+        mimeType = toMimeType format
+        scaleFactor = toNumber state.exportScale / 100.0
+        fileName = (safeName state.chordShape.name) <> "_guitar"
```

Comment:
The export file name should use ```_tenorGuitar``` or ```_tenor_guitar``` not simply ```_guitar```

Reply:
Resolved in bd6ad96903a4afbe14b6bf636ba2739b94aefca4.

Thanks for the review!